packages/js/data/src/payment-settings/types.ts

```diff
@@ -14,10 +14,17 @@ export interface StateData {
 	enabled: boolean;
 	needs_setup: boolean;
 	test_mode: boolean;
+	dev_mode: boolean;
+}
+
+export interface LinkData {
+	href: string;
 }
 
 export interface ManagementData {
-	settings_url: string; // URL to the settings page for the payment gateway.
+	_links: {
```

Comment:
What's the reasoning behind adding `_` to some fields but not others? 🤔

Reply:
The only ones that don't have a underscore are the list of provider links because those are non-standard (have `_type` etc). The rest all follow the WP API patterns of `_links` since they are links for taking some action (are not presentational).